lib/sierra-models/base.js

```diff
@@ -349,6 +351,11 @@ class SierraBase {
           varFieldMatchReturn.value = field.content
         }
 
+        // Trim trailing punctuation:
+        if (varFieldMatchReturn.value) {
+          varFieldMatchReturn.value = trimTrailingPunctuation(varFieldMatchReturn.value)
```

Comment:
this seems like a good relocation. what inspired it?

Reply:
Saw a duplicate note come through because our previous de-duping was being applied to the original varfield values and only later trimming trailing punctuation. This meant that trailing punctuation caused two different varfield values to appear as unique, but then later appear identical because of the late application of trimTrailingPunctuation. By trimming before de-duping, we fix this.